website/versioned_docs/version-2.11/api-packages.json

```diff
@@ -1 +1 @@
-[{"entryPoints":{"index":{"label":"Index","path":"src/index.ts"}},"packageRoot":"/Users/adamek/htdocs/apify/apify-client-js","packagePath":".","packageSlug":".","packageName":"apify-client","packageVersion":"2.11.2"}]
\ No newline at end of file
+[{"entryPoints":{"index":{"label":"Index","path":"src/index.ts"}},"packageRoot":"/home/jindrichbar/Desktop/apify/apify-client-js","packagePath":".","packageSlug":".","packageName":"apify-client","packageVersion":"2.11.0"}]
```

Comment:
huh, how come this is an absolute path. the old snapshots have all relative path in here

Reply:
Damn, could this be the issue? I've been trying to make it work the whole afternoon, no dice... then I regenerated the snapshots and it started working.

The true definition of _it worked on my computer_.

Reply:
(btw removing the `packageRoot` field manually or replacing it with a relative path did not help).